warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/IngestJob.java

```diff
@@ -338,6 +338,12 @@ public int run(String[] args) throws Exception {
                 configureBulkPartitionerAndOutputFormatter(job, cbHelper, conf, outputFs);
             } catch (Exception e) {
                 log.error(e);
+                log.info("Deleting orphaned directory: " + workDirPath);
+                try {
+                    inputFs.delete(workDirPath, true);
+                } catch (Exception er) {
+                    log.error("Unable to remove directory: " + workDirPath);
```

Comment:
Sorry, missed this on the first pass. Mind if we add the er to the log? That'll show us the "Caused By:" stacktrace.

```suggestion
                    log.error("Unable to remove directory: " + workDirPath, er);
```